src/openApi/openApiWrapper.ts

```diff
@@ -29,6 +29,11 @@ export default class OpenApiWrapper {
   private newrelic;
 
   constructor(requestLogger, config?: OpenApiWrapperConfig) {
+    if (config?.enableNewRelicTracking) {
+      // eslint-disable-next-line global-require
+      this.newrelic = require('newrelic');
```

Comment:
Now that this is no longer "async", I could move it to the constructor. For lambdas though, this likely makes no difference